.github/actions/jlab-a11y-test/action.yml

```diff
@@ -0,0 +1,116 @@
+      shell: bash
+
+    - name: Cache conda 🧠
+      uses: actions/cache@v2
+      env:
+        # Increase this value to reset cache if environment.yml has not changed
+        CACHE_NUMBER: 0
+      with:
+        path: ~/conda_pkgs_dir
+        key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{
+          hashFiles('environment.yml') }}
+
+    - name: Install miniconda 🐍
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        miniforge-variant: Mambaforge
+        miniforge-version: latest
+        use-mamba: true
+        environment-file: ./testing/jupyterlab/environment.yml
+        activate-environment: a11y-tests
+        use-only-tar-bz2: true
+
+    - name: Sanity checks on environment 🔍
+      run: |
+        conda info
+        conda env list
+        echo $CONDA_PREFIX
+      shell: bash -l {0}
+
+    - name: Checkout JupyterLab
+      uses: actions/checkout@v3
+      with:
+        repository: ${{ inputs.repository || 'jupyterlab/jupyterlab' }}
+        ref: ${{ inputs.ref || 'master' }}
+        path: jupyterlab
+
+    - name: Build JupyterLab
+      run: |
+        conda run --prefix $CONDA_PREFIX ./scripts/ci_install.sh
+        conda run --prefix $CONDA_PREFIX jlpm run build
+      shell: bash -l {0}
+      working-directory: jupyterlab
+
+    - name: Check JupyterLab installation
+      run: |
+        which jupyter
+        jupyter lab --version
+      shell: bash -l {0}
+
+    - name: Install node dependencies 🧶
+      run: |
+        conda run --prefix $CONDA_PREFIX npm install
+        conda run --prefix $CONDA_PREFIX npx playwright install --with-deps chromium
+      shell: bash -l {0}
+      working-directory: testing/jupyterlab
+
+    - name: Run tests  ✅
+      continue-on-error: true
+      run: |
+        conda run --prefix $CONDA_PREFIX npm test
+      shell: bash -l {0}
+      working-directory: testing/jupyterlab
+
+    - name: Write GitHub Summary
+      run: cat test-results/jupyterlab-a11y-regression-test-results.md >> $GITHUB_STEP_SUMMARY
+      shell: bash -l {0}
+      working-directory: testing/jupyterlab
+
+    - name: Get run refs 🏷
+      id: getrefs
+      run: |
+        export raw_branch=${GITHUB_REF#refs/heads/}
+        echo "branch=${raw_branch//\//-}" >> $GITHUB_OUTPUT
+        echo "sha8=$(echo ${GITHUB_SHA} | cut -c1-8)" >> $GITHUB_OUTPUT
+      shell: bash -l {0}
+
+    - name: Upload Playwright report 📤
+      uses: actions/upload-artifact@v3
+      # using default retention policy = 90 days
+      with:
+        name: playwright-report-${{ steps.getrefs.outputs.branch }}-${{ steps.getrefs.outputs.sha8 }}
+        path: ./testing/jupyterlab/playwright-report/
+
+    - name: Upload JSON test results 📑
+      uses: actions/upload-artifact@v3
+      with:
+        name: json-results-test-a11y-${{ steps.getrefs.outputs.branch }}-${{ steps.getrefs.outputs.sha8 }}
+        path: ./testing/jupyterlab/test-results/jupyterlab-a11y-regression-test-results.json
```

Comment:
I think it would be good to create a single zip file with a README. I'll do that in a follow-up:

- https://github.com/Quansight-Labs/jupyter-a11y-testing/issues/26

Reply:
- https://github.com/Quansight-Labs/jupyter-a11y-testing/issues/26